Kitodo/src/main/java/org/kitodo/services/data/ProjectService.java

```diff
@@ -282,4 +283,62 @@ public boolean isProjectComplete(Project project) {
         return project.getTitle() != null && project.template != null && project.getFileFormatDmsExport() != null
                 && project.getFileFormatInternal() != null && digitalCollectionsXmlExists && projectsXmlExists;
     }
+
+    /**
+     * Duplicate the project with the given ID 'itemId'.
+     *
+     * @return the duplicated Project
+     */
+    public Project duplicateProject(Integer itemId) throws DAOException {
+        Project duplicatedProject = new Project();
+
+        Project baseProject = getById(itemId);
+
+        // Project _title_ should explicitly _not_ be duplicated!
```

Comment:
How did you find the project after saving again if it has no title? Should the user not insert a new project title on starting duplicating a project and this inserted title is set here?

Reply:
The `title` field is empty when the user duplicates a project. Since the `title` field is mandatory, the user cannot save the duplicated project without entering a new title.

Reply:
OK if the user get the duplicated project displayed and must enter the project title, then it should work.